### ISSUE TITLE
Rancher CLI should provide an appropriate error message for invalid/unsupported -e option

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#9132]
**Rancher versions:**
rancher/server:v1.6.3-rc1

**Rancher CLI version**
v0.6.2-rc1

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**Ubuntu

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**GCE

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single node

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**Cattle

**Steps to Reproduce:**
1. Set up rancher CLI 
2. Start a container with -e "=test" which is not supported

**Results**
CLI just prints the container id.  The container fails to start and errors out in the setup
CLI should provide a valid error message that the option -e"=test" is not supported or it is invalid similar to docker 

```
soumyas-MBP:rancher-v0.6.2-rc1 soumya$ ./rancher run -e "=test" -i --name testcontainer1 ubuntu 
1s95

```

**Comment by carolynhu** [comment]
Available in CLI v0.6.2-rc2

**Comment by soumyalj** [comment]
Tested with CLI v0.6.2-rc2 using rancher v1.6.3-rc1.  Starting a container with -e "=foo1" option gives an appropriate error message. The container does not error out in the setup. 
```
soumyas-MBP:rancher-v0.6.2-rc2 soumya$ ./rancher run -e "=foo1" --name test1 -i ubuntu
invalid argument "=foo1" for e: invalid environment variable: =foo1
See 'rancher run --help'.

```
